### PR TITLE
Removes flickr from repo, adds migration for db

### PR DESCRIPTION
**Pull request by jduss4**

### CODE REVIEW

File: core/templates/reports/reports.html [hunk 1]
```diff
@@ -3,18 +3,15 @@
 {% block subcontent %}
 
   <table class="data table table-striped table-hover" width="100%">
-    <tr class='lightGray'>
+    <tr class='white'>
```
Comment (jechols) [inline core/templates/reports/reports.html]: This seems pretty unrelated to flickr changes

Comment (jechols) [inline core/templates/reports/reports.html]: Okay, I see what's going on, hah

Comment (jduss4) [inline core/templates/reports/reports.html]: I don't want to talk about it. :'(
